### ISSUE TITLE
Streaming region toggling

### ISSUE THREAD

**Issue by pafade89** [Fallenbagel/jellyseerr#1200]
### Description

The new setting is toggling.

https://github.com/user-attachments/assets/aff94286-1b46-460c-9246-8e60f0e7e600

### Version

2.2.2

### Steps to Reproduce

![image](https://github.com/user-attachments/assets/d28f17a3-176d-442c-9f23-4a15d8da3853)

### Platform

desktop

### Database

None

### Device

PC

### Operating System

Windows 11

### Browser

Chrome

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by tmsholy** [comment]
Can confirm this is also happening for me on `2.2.2`. Just to clarify this is for the global settings, the per-user setting does not seem to do this.

**Comment by Fallenbagel** [comment]
> Can confirm this is also happening for me on `2.2.2`. Just to clarify this is for the global settings, the per-user setting does not seem to do this. 

Yeah those two gets values differently. Global gets from settings.json and the other from db
